zebra-rpc/src/methods/types/transaction.rs

```diff
@@ -564,6 +569,7 @@ impl TransactionObject {
             },
             size: tx.as_bytes().len().try_into().ok(),
             time: block_time.map(|bt| bt.timestamp()),
+            txid: tx.hash(),
```

Comment:
Could we accept this as a parameter to this function instead? The `getrawtransaction` method accepts the transaction hash as a parameter, and I think it's used by the lightwalletd, so hashing every transaction during the lwd sync may cause a performance regression.

Reply:
Done in https://github.com/ZcashFoundation/zebra/pull/9650